cmake/Modules/SWIG_CGAL_Macros.cmake

```diff
@@ -16,16 +16,19 @@ MACRO(ADD_SWIG_CGAL_LIBRARY libname)
   include_directories(${CMAKE_CURRENT_SOURCE_DIR})
   set(CMAKE_LIBRARY_OUTPUT_DIRECTORY "${COMMON_LIBRARIES_PATH}")
   set(CMAKE_RUNTIME_OUTPUT_DIRECTORY "${COMMON_LIBRARIES_PATH}")
-  EXTRACT_CPP_AND_LIB_FILES(${ARGN}) 
+  EXTRACT_CPP_AND_LIB_FILES(${ARGN})
   add_library(${libname} SHARED ${source_files})
   target_link_libraries(${libname} ${libstolinkwith})
+  if(APPLE)
+    set_target_properties(${libname} PROPERTIES LINK_FLAGS "-undefined dynamic_lookup")
+  endif()
```

Comment:
it's only needed for swig modules, not these sublibs

Reply:
Yeah, I noticed that later. So, this is recommended to be added _strictly_ when linking with Python, right?

Reply:
anything that's a plugin